brd-android/rockwallet-common/src/main/java/com/rockwallet/common/utils/Extensions.kt

```diff
@@ -91,19 +90,16 @@ fun Editable?.textOrEmpty() = if (isNullOrEmpty()) "" else toString()
 
 fun Editable?.asInt() = if (isNullOrEmpty()) null else toString().toInt()
 
-fun TextInputLayout.setEnabledView(enabled: Boolean) = run {
+fun TextInputLayout.setEnabledView(enabled: Boolean, hasChevron: Boolean = false) = run {
     isEnabled = enabled
-    alpha = if (isEnabled) { 1f } else { 0.3f }
-}
-
-fun TextInputEditText.setEnabledView(enabled: Boolean) = run {
-    isEnabled = enabled
-    alpha = if (isEnabled) { 1f } else { 0.3f }
-}
+    alpha = if (isEnabled) { 1f } else { 0.4f }
 
-fun min(a: BigDecimal, b: BigDecimal) = when {
-    b < a -> b
-    else -> a
+    if (hasChevron) {
+        endIconMode = if (enabled) TextInputLayout.END_ICON_CUSTOM else TextInputLayout.END_ICON_NONE
+        if (enabled) {
+            setEndIconDrawable(R.drawable.ic_chevron_down)
+        }
+    }
```

Comment:
What if `hasChevron` changes in the runtime? Maybe do like this:

```suggestion
    if (hasChevron && enabled) {
        endIconMode = TextInputLayout.END_ICON_CUSTOM
        setEndIconDrawable(R.drawable.ic_chevron_down)
    } else {
        endIconMode = TextInputLayout.END_ICON_NONE
    }
```